spec/src/main/asciidoc/overview.adoc

```diff
@@ -27,17 +27,18 @@ https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-read
 
 In this scenario health checks are used to determine if a computing node needs to be discarded (terminated, shutdown) and eventually replaced by another (healthy) instance.
 
-The MP-HC architecture consists of a single `/health` endpoint in a MicroProfile runtime that represents the status of
-the entire runtime. This `/health` endpoint is expected to be associated with a configurable context, such as a web
+The MP-HC architecture consists of two `/health/ready` and `/health/live` endpoint in a MicroProfile runtime that respectively represents the readiness and the status of the entire runtime.
```

Comment:
the status ... -> the liveness status of the entire runtime correspondingly.

Reply:
or replace "status" with "liveness"

Reply:
yep. That will do.

Reply:
I also have to mention backward compatibility with `/health`